lib/transform.js

```diff
@@ -168,3 +169,23 @@ function mapCheckItemIssueToMkdn(issue) {
   }
   return `- [${checked}] #${issue.number}`;
 }
+
+Transform.trelloUsernameToId = function (username) {
+  if (!username) {
+    return;
+  }
+  let ownerInfo = memberlist.find(function (member) {
+    return member.username.includes(username.toLowerCase().slice(1));
+  });
+  return ownerInfo.id;
+};
+
+Transform.trelloIdToUsername = function (id) {
+  if (!id) {
+    return;
+  }
+  let username = memberlist.find(function (member) {
+    return member.id.includes(id);
```

Comment:
I think you mean `member.id === id`. `member.id.includes(id)` would mean that `"abc123".includes("c12")` would match, which is probably not correct.